pgmgr/pgmgr.go

```diff
@@ -279,6 +295,59 @@ type execer interface {
 }
 
 func applyMigration(c *Config, m Migration, direction int) error {
+	if c.MigrationDriver == "psql" {
+		return applyMigrationByPsql(c, m, direction)
+	}
+
+	return applyMigrationByPq(c, m, direction)
+}
+
+func applyMigrationByPsql(c *Config, m Migration, direction int) error {
+	if err := c.DumpToEnv(); err != nil {
+		return err
+	}
+
+	contents, err := ioutil.ReadFile(filepath.Join(c.MigrationFolder, m.Filename))
+	if err != nil {
+		return err
+	}
+
+	tmpfile, err := ioutil.TempFile("", "migration")
+	if err != nil {
+		return err
+	}
+	defer os.Remove(tmpfile.Name()) // clean up
+
+	if m.WrapInTransaction() {
```

Comment:
Another option with `psql` is to simply add the command line flag `-1`, which wraps the file in a transaction.